draft-ietf-openpgp-pqc.md

```diff
@@ -451,13 +494,13 @@ The operation `x25519Kem.Encaps()` is defined as follows:
 
  3. Set the output `ecdhCipherText` to `V`
 
- 4. Set the output `ecdhKeyShare` to `SHA3-256(X || ecdhCipherText || ecdhPublicKey)`
+ 4. Set the output `ecdhKeyShare` to `X`
```

Comment:
We could simply say above
`ecdhKeyShare = X25519(v, R)`.
That saves one line in each algorithm description just to assign X. But maybe we want to keep `X` for clarity?

Reply:
I think I would prefer it as it is right now, the first steps describe the computational steps and the last "Set" statements take the role of a "return".